inference-engine/src/low_precision_transformations/src/weightable_layer_transformation.cpp

```diff
@@ -115,10 +115,18 @@ bool WeightableLayerTransformation::canBeTransformed(const TransformationContext
             return false;
         }
 
-        if ( // Check if all dimensions of scale except the first one (which is O-Output channels dimension) are all ones
-            (shape_size(constOutputShape) != constOutputShape[0]) ||
-            ((constOutputShape[0] != 1ul) && (fqFromWeights->get_output_shape(0)[0] != constOutputShape[0]))) {
-            return false;
+        if (!is_type<opset1::ConvolutionBackpropData>(layer)) {
+            if ( // Check if all dimensions of scale except the first one (which is O-Output channels dimension) are all ones
+                (shape_size(constOutputShape) != constOutputShape[0]) ||
+                ((constOutputShape[0] != 1ul) && (fqFromWeights->get_output_shape(0)[0] != constOutputShape[0]))) {
+                return false;
+            }
+        } else {
+            if ( // Check if all dimensions of scale except the second one (which is O-Output channels dimension) are all ones
+                (shape_size(constOutputShape) != constOutputShape[1]) ||
+                ((constOutputShape[1] != 1ul) && (fqFromWeights->get_output_shape(0)[1] != constOutputShape[1]))) {
+                return false;
+            }
```

Comment:
is it copy/paste? can we specify `outChannelsShapeIndex` and reuse?
```cpp
const size_t outChannelsShapeIndex = is_type<opset1::ConvolutionBackpropData>(layer) ? 1ul : 0ul;
if ( // Check if all dimensions of scale except the second one (which is O-Output channels dimension) are all ones
    (shape_size(constOutputShape) != constOutputShape[outChannelsShapeIndex]) ||
    ((constOutputShape[outChannelsShapeIndex] != 1ul) && (fqFromWeights->get_output_shape(0)[outChannelsShapeIndex] != constOutputShape[outChannelsShapeIndex]))) {
    return false;
}
```

Reply:
resolved in #4764